examples/count-hosts.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2008 the V8 project authors. All rights reserved.
```

Comment:
Remove this file - it isn't used anywhere?

Reply:
This is included in `v8/samples`, and this intended to be used with `process.cc` (`process.rs`), which will not run without any script.